pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -31,6 +31,7 @@ import (
 const (
 	machineAnnotationKey = "machine.openshift.io/machine"
 	ownerControllerKind  = "MachineSet"
+	disableFencingLabel = "machine.openshift.io/healthcheck-disabled"
```

Comment:
if anything at all this should be an annotation

Reply:
@emesika ack. we'll also change to healthchecking.openshift.io/disabled

Do you have any other concerns about the concept in general?

Reply:
this will also need unit test. Can you please share a gdoc with all the rationale annotations you think you need, use cases and scenarious for the baremetal story?
If you want to opt out why don't just remove that machine from the MHC target or remove the MHC resource?

Reply:
Is there any reason why this can't be in the remediation controller for now?

Reply:
Generally its better that the work is not scheduled if we don't want it to take place. Otherwise it looks like there is work to do when there isn't, which can look messy and confuse admins.